README.md

```diff
@@ -2342,6 +2345,36 @@ plist-like key-value pairs, similarly to &keys keyword of
 This binds `n` values from the list to a1 ... aN, then interprets
 the cdr as a plist (see key/value matching above).
 
+`a` shorthand notation for kv-destructuring exists which allows the
```

Comment:
I think the first letter here shouldn't be in backticks.

Reply:
Yea, I've noticed this too.  The problem is that the sentence starts with `A` which the script thinks is a variable :D And so fontifies it accordingly.  I think I'll need to reorder the sentence to start with a regular word.